src/components/Navbar.tsx

```diff
@@ -1,37 +1,29 @@
 "use client";
 
 import Image from "next/image";
-
-
-
 import { useState } from "react";
 import Link from "next/link";
 import { Equal, X } from "lucide-react";
 import { Button } from "@/components/ui/button";
+import PromochatorLogo from "../../public/assets/logo/promochator-logo.svg"
 
 export function Navbar() {
   const [showSidebar, setShowSidebar] = useState<boolean>(false);
   return (
     <>
-      <div className="w-full flex flex-row justify-between items-center gap-12">
-        <div className="flex flex-row gap-2 items-center mr-auto">
-          <Image
-            src="/assets/logo/promochator_logo.png"
-            alt="Promochator Logo"
-            className="w-6"
-            width={150}
-            height={150}
-          />
-          <p className="text-lg font-semibold">
-            Promo<span className="text-[#5f7ecd]">CHAT</span>or
-          </p>
-        </div>
+      <div className="flex justify-between items-center">
+        <Image
+          src={PromochatorLogo}
+          alt="logo Promochatora"
+          width={160}
+          height={34}
+        />
```

Comment:
bardziej jako protip, lubię logo eksportować sobie w png i wtedy nie trzeba width i height podawać, bo nextjs sam sobie to ogarnia

na stronach internetowych też czasami png mniej waży niż svg, a różnicy w jakości nikt nie widzi 😭

Reply:
logo w .svg waży 6.44KB a w .png 1.87KB, osobiście uważam że jest widoczna różnica w jakości - .png jest zdecydowanie lekko zblurowane, myślę że nie jest to aż taka różnica w rozmiarze żeby podmieniać, a jakość jest lepsza

Reply:
no spoko, po prostu, żebyś wiedział, że używanie png to też jest poprawna opcja, bo sporo osób się fiksuje na tym, że logo = svg i zapomina o tym, że można inaczej